task-4/task-4/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace task_4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello, user!");
+            Console.WriteLine($"Your answer is {CalculateFullExpression()}"); ;
+            Console.WriteLine("/// type to exit");
+            Console.ReadKey();
+        }
+
+        static double CalculateFullExpression()
+        {
+            double answer = CalculateSmallerPart(3, 5) * CalculateSmallerPart(3, 7) * CalculateSmallerPart(5, 5);
+            return Math.Round(answer, 2);
```

Comment:
Нужно было округлять до 3-х знаков после запятой.
```suggestion
            Console.WriteLine($"Your answer is {CalculateFullExpression()}"); ;
            Console.WriteLine("/// type to exit");
            Console.ReadKey();
        }

        static double CalculateFullExpression()
        {
            double answer = CalculateSmallerPart(3, 5) * CalculateSmallerPart(3, 7) * CalculateSmallerPart(5, 5);
            return Math.Round(answer, 3);
```
А лучше, чтобы метод не округлял, а просто возвращал вычисленное значение. Округление лучше сделать на этапе форматирования при выводе на консоль. Ведь в задании

> Результат выведите на консоль с точностью до трех знаков после запятой.

То есть выведите в заданном формате, не округлите значение.
```suggestion
            Console.WriteLine($"Your answer is {CalculateFullExpression():F3}"); ;
            Console.WriteLine("/// type to exit");
            Console.ReadKey();
        }

        static double CalculateFullExpression()
        {
            double answer = CalculateSmallerPart(3, 5) * CalculateSmallerPart(3, 7) * CalculateSmallerPart(5, 5);
            return answer;
```